src/settings/target/manifest.rs

```diff
@@ -46,7 +48,16 @@ impl Manifest {
     pub fn new(config_path: &Path) -> Result<Self, failure::Error> {
         let config = read_config(config_path)?;
 
-        let manifest: Manifest = config.try_into()?;
+        let manifest: Manifest = match config.try_into() {
+            Ok(m) => m,
+            Err(e) => {
+                if e.to_string().contains("unknown field `kv-namespaces`") {
```

Comment:
if there's a nicer way to match on the specific error thrown by `serde` when the `#[serde(deny_unknown_fields)]` condition is triggered, I'm all ears!